tasks/system_probe.py

```diff
@@ -797,7 +797,7 @@ def kitchen_prepare(ctx, windows=is_windows, kernel_release=None, ci=False, pack
         if pkg.endswith("java"):
             shutil.copy(os.path.join(pkg, "agent-usm.jar"), os.path.join(target_path, "agent-usm.jar"))
 
-        for gobin in ["gotls_client", "fmapper", "prefetch_file"]:
+        for gobin in ["gotls_client", "external_server", "fmapper", "prefetch_file"]:
```

Comment:
what is an external_server? is it a general purpose server or is it only for grpc? in the latter case, I suggest to include the "grpc" prefix in the name